tao/tao_bridge/passes/tao_mark_for_compilation_pass.cc

```diff
@@ -954,7 +954,7 @@ Status MarkForCompilationPassImpl::CreateClusters() {
       string& name = cluster_names[cluster->cycles_graph_node_id()];
 
       if (name.empty()) {
-        name = absl::StrCat("cluster_", GetNextClusterSequenceNumber());
+        name = absl::StrCat("tao_cluster_", GetNextClusterSequenceNumber());
```

Comment:
disc_cluster?

Reply:
I just follow the convention. All other variables and even the filename have 'TAO' as prefix.

Reply:
I see, maybe we should rename them to `DISC` prefix in the future.